hack/testing/init-log-stack

```diff
@@ -31,7 +32,7 @@ openshift_logging_master_public_url=https://${PUBLIC_MASTER_HOST:-localhost}:844
 
 openshift_logging_image_prefix=$imageprefix
 openshift_logging_use_ops=${ENABLE_OPS_CLUSTER:-False}
-openshift_logging_fluentd_use_journal=${USE_JOURNAL:-}
+$USE_JOURNAL_VAR
```

Comment:
If `USE_JOURNAL` is explicitly set, then explicitly set `openshift_logging_fluentd_use_journal=$USE_JOURNAL` in the inventory.
If `USE_JOURNAL` is not set, then omit `openshift_logging_fluentd_use_journal` from the inventory, so that we will exercise the ansible code paths where `openshift_logging_fluentd_use_journal` is not defined.